style.css

```diff
@@ -0,0 +1,220 @@
+main {
+  display: grid;
+  gap: 2px;
+  grid-template-columns: repeat(4 1fr);
+  grid-template-areas:
+    "main main main main"
+    "one two three four";
+  width: 1440px;
+  margin: 0 auto;
+}
+
+.main {
+  background: url(images/main-img.jpg) 50% 100% no-repeat;
+  height: 650px;
+  grid-area: main;
+}
+
+.one {
+  grid-area: one;
+  height: 200px;
+  background: url(images/image-one.jpg);
+}
+
+.two {
+  grid-area: two;
+  background: url(images/image-two.jpg);
+}
+
+.three {
+  grid-area: three;
+  background: url(images/image-three.jpg);
+}
+
+.four {
+  grid-area: four;
+  background: url(images/image-four.jpg);
+}
+
+nav {
+  position: absolute;
+  left: 50%;
+  display: flex;
+  width: 980px;
+  height: 36px;
+  background:
+    linear-gradient(
+      rgba(138, 138, 138, 1) 0%,
+      rgba(123, 123, 123, 1) 26%,
+      rgba(115, 115, 115, 1) 47%,
+      rgba(95, 95, 95, 1) 48%,
+      rgba(99, 99, 99, 1) 77%,
+      rgba(111, 111, 111, 1) 100%
+    );
+  text-align: center;
+  border-radius: 10px;
+  margin-top: 100px;
+  transform: translate(-50%, -50%);
+}
+
+.apple {
+  position: relative;
+  top: 20%;
+  height: 21px;
+}
+
+nav a,
+nav div {
+  display: inline-block;
+  flex-grow: 1;
+  box-shadow: 0.5px 0 0.1px 0.04px #949494;
+  vertical-align: middle;
+  position: relative;
+}
+
+nav a {
+  border-right: 0.02px solid #575757;
+}
+
+.glass {
+  position: absolute;
+  top: 28%;
+  left: 22%;
+  width: 13px;
+}
+
+nav input {
+  position: relative;
+  top: 20%;
+  width: 100px;
+  height: 19px;
+  background: transparent;
+  box-sizing: border-box;
+  border-radius: 12px;
+  border: 1px solid #585858;
+  transition: width 0.3s ease-in-out;
+  padding: 2px 18px 1px 25px;
+  box-shadow: 0 -2px black, 0 0.5px white;
+}
+
+nav span {
+  position: relative;
+  top: 20%;
+  color: white;
+}
+
+h1,
+h2,
+p {
+  position: absolute;
+  color: white;
+  text-align: center;
+}
+
+h1 {
+  left: 50%;
+  transform: translate(-50%, -50%);
+  top: 250px;
+  font-size: 60px;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 20px;
+  font-weight: 100;
+}
+
+h2 {
+  left: 50%;
+  transform: translate(-50%, -50%);
+  top: 350px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  line-height: 30px;
+  font-weight: 400;
+}
+
+p {
+  left: 50%;
+  transform: translate(-50%, -50%);
+  top: 450px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 25px;
+  line-height: 18px;
+  font-weight: 100;
+}
+
+p img {
+  height: 25px;
+}
+
+footer {
+  width: 1000px;
+  margin: 10px auto 0;
+}
+
+.second-footer {
+  display: inline-block;
+  margin-left: 90px;
+  font-weight: 400;
+}
+
+.bottom-footer {
+  display: block;
+  margin-top: 10px;
+}
+
+footer img {
+  position: relative;
+  top: 6px;
+}
+
+footer span {
+  color: #6e6e6e;
+  font-size: 10px;
+  font-family:
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Helvetica,
+    Arial,
+    Verdana,
+    sans-serif;
+  line-height: 12px;
+  font-weight: 400;
```

Comment:
this it seems it has no effect since 400 is the same as normal thickness, you can remove it